--- a/conf/local.conf	2019-08-08 10:59:57.043695699 +0800
+++ b/conf/local.conf	2019-08-02 17:40:32.785393847 +0800
@@ -43,7 +43,7 @@
 # You need to select a specific machine to target the build with. These are the
 # machines which for the Toradex Colibri and Apalis computer on module families
 #
-#MACHINE ?= "apalis-imx6"
+MACHINE ?= "apalis-imx6"
 #MACHINE ?= "apalis-t30"
 #MACHINE ?= "apalis-tk1"
 #MACHINE ?= "apalis-tk1-mainline"
@@ -52,7 +52,7 @@
 #MACHINE ?= "colibri-imx6ull"
 #MACHINE ?= "colibri-imx7"
 #MACHINE ?= "colibri-imx7-emmc"
-MACHINE ?= "colibri-t20"
+#MACHINE ?= "colibri-t20"
 #MACHINE ?= "colibri-t30"
 #MACHINE ?= "colibri-vf"
 
@@ -275,3 +275,11 @@
 # This file does not need to exist, if it does it can be used to influence machine specific
 # configurations without copying the machine file.
 include conf/machine/include/${MACHINE}.inc
+
+IMAGE_INSTALL_append = " cryptsetup cryptodev-module cryptodev-tests lvm2 rng-tools"
